doc/src/base/multi-threading.md

```diff
@@ -1,15 +1,23 @@
 # [Multi-Threading](@id lib-multithreading)
 
-This experimental interface supports Julia's multi-threading capabilities. Types and functions
-described here might (and likely will) change in the future.
-
 ```@docs
-Base.Threads.threadid
-Base.Threads.nthreads
 Base.Threads.@threads
 Base.Threads.@spawn
+Base.Threads.threadid
+Base.Threads.nthreads
 ```
 
+## Synchronization
+
+```@docs
+Base.Threads.Condition
+Base.Threads.Event
+```
+
+See also [Synchronization](@ref lib-task-sync).
+
+## Atomic operations
+
```

Comment:
I think we should mark these as subject to possible change.

Reply:
Ok. Do you think we'd need to delete them entirely, or just add a better interface?

Reply:
Yeah, just don’t want to commit exactly to these yet